Documentation/tw_docs.md

```diff
@@ -55,4 +55,4 @@ This source code is released under the GPL 3.0 license. Please read the license
 ## Getting Started
 Whether you are a developer looking to extend the functionality of the smartwatch or a user curious about the technology behind the device, this documentation is your gateway to unlocking the full potential of the TinyWATCH S3 firmware.
 
-Let's embark on this journey of exploration and innovation as we delve into the heart of the TinyWATCH S3 firmware.
\ No newline at end of file
+Let's embark on this journey of exploration and innovation as we delve into the heart of the TinyWATCH S3 firmware.
```

Comment:
Huh. That's annoying. It seems GitHub's "in-place" editing will append a new-line at the end of the file.

Happy to revert if that's undesired.

Reply:
There's supposed to be an empty new line at the end. That's common practice.

Reply:
I agree, I'm just surprised the GitHub editor did so simply by touching the file. Though I sometimes wish there was a standard `git smudge` for it on text files.